web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -106,85 +183,6 @@
     </div>
   </div>
 </template>
-
-<script>
```

Comment:
Now I'm confused.
How did the diff algorithm suddenly manage to move instead of the template block?

Reply:
Maybe this move detection is based on number of lines and it moves the one with less lines?